src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -283,11 +308,20 @@ def _compute_metrics(self, sorting_analyzer, unit_ids=None, verbose=False, **job
                 template_metrics.at[index, metric_name] = value
         return template_metrics
 
-    def _run(self, verbose=False):
+    def _run(self, delete_existing_metrics=False, verbose=False):
         self.data["metrics"] = self._compute_metrics(
             sorting_analyzer=self.sorting_analyzer, unit_ids=None, verbose=verbose
         )
 
+        tm_extension = self.sorting_analyzer.get_extension("template_metrics")
```

Comment:
This makes me think actually that the previously computed metrics are not recomputed. But if so I have two questions:

1) where is that logic to avoid in `_compute_metrics`?
2) at this stage, is `self.sorting_analzyer.get_extension("template_metrics")` an old version of the template metrics object that is due to be overwritten by the current one, but has not been yet? so we can pass the previously computed metrics from this instantiation to the current one?

If so that's very nice, a couple of lines to this function docstring to explain I think would make this clear.